modules/ecs_cluster/main.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_ecs_cluster" "this" {
+  name = "${var.namespace}-ecs-cluster"
+
+  setting {
+    name  = "containerInsights"
+    value = "enabled"
+  }
+}
```

Comment:
Do we need `aws_ecs_cluster` to be in a separate module instead of including it inside `aws_ecs_service ` in module `ecs` ?

Reply:
I think separating could be easier to reach, the aws_ecs module is already huge with loads configs